perceval/backends/_mps.py

```diff
@@ -144,8 +143,6 @@ def _compile(self) -> bool:
         # This initialization of MPS (gamma and sv) fixes the input state to be completely separable
         # and a pure BasicState (no superposition); hence would have only 1 non-zero element whose value = 1.
         # It is simply written based on this choice as the SVD of such a structure would exactly look like this
-        # todo: initialization of MPS could be more generic - to include mixed/superposed states as input
```

Comment:
Did we drop this idea ? If not is there a JIRA ticket that exists ?

Reply:
I updated this as a comment on the current JIRA ticket. We need to decide if we want to create tickets now or keep this information elsewhere